CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Changed
 
 - AI data is synchronizable between devices via the new functions `syncdata_bytes` and `synchronize`.
+- Add multi-threading support: Parallelism is always enabled for mobile targets and can optionally be enabled for web targets. A `FeatureHint` is now required to pick the assets for the chosen target during the setup (see `getInputData()`), available choices are `mobileParallel`, `webSequential` and `webParallel`. The asset generation and publishing has been updated accordingly.
```

Comment:
If this is related to TY-2066, so maybe we can sort this in there, but I'll prefer to not have `mobileParallel` and have
something like `getInputData([List<FeatureHint> = const []]). The mobile ignore it while the web look what is inside.

Since we will need more hints in the future maybe is better to have only the positive version of it as an hint and considered that is not available if not present.

Reply:
i'll make the required changes in #202 and then update the changelog.